CMakeLists.txt

```diff
@@ -98,6 +98,12 @@ If(NOT _HAS_CXX11_FLAG)
 EndIf()
 
 # Load some basic macros which are needed later on
+find_package(FairCMakeModules 0.1 CONFIG)
```

Comment:
Why is the `CONFIG` needed?

Reply:
I found the verbosity about find-modules annoying. As we now have a dedicated warning (see below), a QUIET is probbaly better?